src/fs/driver/fat/fat_common.c

```diff
@@ -252,18 +273,18 @@ uint32_t fat_get_volinfo(void *bdev, struct volinfo * volinfo, uint32_t startsec
 	volinfo->numsecs =  (uint16_t) lbr->bpb.sectors_s_l |
 		  (((uint16_t) lbr->bpb.sectors_s_h) << 8);
 
-	if (!volinfo->numsecs) {
```

Comment:
It's not really neccessary, m.b. it will just make code more readable. Still ok for me.